rails/config/routes.rb

```diff
@@ -6,6 +6,7 @@
   # root 'welcome#index'
   namespace :api do
     resources :posts
+    resources :analytics
```

Comment:
You may wonder why this is necessary. The main reason from a development standpoint is that it keeps your `rake routes` output tidy. Also, if you have extra routes set up without corresponding controller actions, if someone goes to that URL it'll throw a 500 instead of a 404.